addons/point_of_sale/models/pos_payment.py

```diff
@@ -10,6 +10,7 @@ class PosPayment(models.Model):
     """
 
     _name = "pos.payment"
+    _rec_name = "id"
```

Comment:
After talking with LCA, we would see the Order name and the amount in the breadcrumb. I think it can be done by overriding diplay_name